cernopendata/modules/fixtures/data/records/cms-derived-Run2-datascience.json

```diff
@@ -0,0 +1,386 @@
+{
+  "abstract": {
+    "description": "FIXME"
```

Comment:
See abstract info in https://github.com/cernopendata/opendata.cern.ch/issues/2447#issuecomment-479385835